build.rs

```diff
@@ -188,6 +372,21 @@ fn load_cross_compile_info() -> Result<(InterpreterConfig, HashMap<String, Strin
     Ok((interpreter_config, fix_config_map(config_map)))
 }
 
+#[allow(unused_variables)]
+fn load_cross_compile_info(
+    python_paths: CrossPython,
+) -> Result<(InterpreterConfig, HashMap<String, String>)> {
+    let target_family = env::var("CARGO_CFG_TARGET_FAMILY")?;
+    // Because compiling for windows on linux still includes the unix target family
+    if target_family == "unix" {
+        // Configure for unix platforms using the sysconfigdata file
+        load_cross_compile_from_sysconfigdata(python_paths)
+    } else {
+        // Must configure by headers on windows platform
+        load_cross_compile_from_headers(python_paths)
```

Comment:
I'd also be happy for us to just hard-code an implementation for windows which is inspired by `sysconfig.py`'s non-posix function: https://github.com/python/cpython/blob/2bcd0fe7a5d1a3c3dd99e7e067239a514a780402/Lib/sysconfig.py#L426-L435

I can do that in a separate PR, if you like. If so let's remember to create an issue once this PR is merged.

Reply:
I think it would be more appropriate in a separate PR. It might seem out of scope since this one is more geared towards using the sysconfigdata. While windows requires a whole other tactic.

Reply:
I'll do it in #923; I'll rebase and improve after this merges.